### PR TITLE
Style The Bundles On The Landing Page, And Apply Design Changes

### PR DESCRIPTION
**Pull request by Ap0c**
## Why are you doing this?

a) The bundles on the landing page need some styling applied to them.
b) There are some updated designs for the bundles, which are applied here.

This is mostly heavy on the CSS, to apply styling to the bundles across the breakpoints, and there a couple of React/JSX tweaks to go with this. The layouts at narrower breakpoints don't quite match what this page looks like in membership-frontend, because some of our breakpoints and margins don't quite match up.

There are still a couple of bugs in IE10 and Safari 8 (sigh), which can be fixed in a later PR when we're sure that we won't be making more design changes before going live.

[**Trello Card**](https://trello.com/c/52DEBiP3/573-style-the-bundles-section-of-the-landing-page), and [another](https://trello.com/c/zpuu7hjY/550-change-the-become-a-digital-subscriber-to-start-your-14-day-trial), and [one more](https://trello.com/c/6W4rSo7L/551-swap-the-headers-round-on-the-benefits).

FYI: @superfrank

### Design Changes

- Swapped the bundle headers around and changed the copy.
- Changed the colour scheme of the contributions toggles to match the CTA.
- Dropped the paper bundle toggle in favour of dual-CTAs.
- Updated the features copy for the digital and paper bundles.
- Simplified the copy in the introduction to bring in line with changes made in [membership-frontend](https://github.com/guardian/membership-frontend/pull/1632).

### General Styling

- Added a right arrow to the bundles CTAs.
- Applied fonts to the bundles.
- Made layouts for the different breakpoints.

## Screenshots

I'll fill this out with screenshots across the breakpoints, but the easiest way to see it is probably just to pull it down and look at it.

### CODE REVIEW

File: assets/components/featureList/featureList.scss [hunk 1]
```diff
@@ -2,8 +2,26 @@
...
+
+	p {
+		font-family: $gu-text-sans-web;
+		font-weight: 300;
+		line-height: 20px;
+	}
+
+	li {
+		margin-bottom: $gu-v-spacing;
+		margin-left: 19px;
```
Comment (svillafe) [inline assets/components/featureList/featureList.scss]: Is it possible to set this to `$gu-h-spacing`? 19 px feels weird.

Comment (Ap0c) [inline assets/components/featureList/featureList.scss]: Yup, I expect so, the design is all a bit arbitrary.

Comment (Ap0c) [inline assets/components/featureList/featureList.scss]: 6234bb2

Comment (svillafe) [inline assets/components/featureList/featureList.scss]: 👍

**Pull request merged**